zerver/lib/markdown/__init__.py

```diff
@@ -134,7 +134,16 @@ class MessageRenderingResult:
     potential_attachment_path_ids: List[str]
 
 
-DbData = Dict[str, Any]
+@dataclass
+class DbData:
+    mention_data: MentionData
+    realm_uri: str
+    realm_alert_words_automaton: Optional[ahocorasick.Automaton]
+    active_realm_emoji: Dict[str, Dict[str, str]]
```

Comment:
This annotation is wrong; it looks like this was propagated from an existing wrong annotation. An emoji dict seems to have `id: str, name: str, source_url: str, deactivated: bool, author_id: int`.

Reply:
I added a commit to address this.  It's slightly annoying due to `still_url`, which I explain in the commit message.